docs/installation.md

```diff
@@ -46,7 +48,7 @@ Execute the following command:
   ```
```

Comment:
This shows to set the conda-forge channel, but the command is different than in the other file. Is that on purpose?

Reply:
Ah good catch, these docs where made quite some time apart. Good to keep them in line indeed (updating the readme, because the command in this file is better to use)